new/LIT1990Mawase.xml

```diff
@@ -0,0 +1,316 @@
+               <label>Antiphon of Michael (<foreign xml:lang="gez">ዘሚካኤል፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4486Garima1">
+               <label>Antiphon of ʾAbbā Garimā (<foreign xml:lang="gez">ዘአባ፡ ገሪማ፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4487Garima2">
+               <label>Antiphon of ʾAbbā Garimā (<foreign xml:lang="gez">ዘአባ፡ ገሪማ፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4488Apostles">
+               <label>Antiphon of the Apostles (<foreign xml:lang="gez">ዘሐዋርያት፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4489Qirqos">
+               <label>Antiphon of Cyriacus (<foreign xml:lang="gez">ዘቂርቆስ፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4490Virgins">
+               <label>Antiphon of the Virgins (<foreign xml:lang="gez">ዘደናግል፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4491Firstborn">
+               <label>Antiphon of the Firstborn (<foreign xml:lang="gez">ዘማኅበረ፡ በኵር፡</foreign>)</label>
+               <!-- In BAVet15#p1_i9 an antiphon "in hiemali congregatione (ዘማኅበር፡ ዘክረምት፡)" -->
+            </div>
+            <div type="textpart" corresp="LIT4492DabraTabor">
+               <label>Antiphon of Mount Tabor, i.e. the Transfiguration of Jesus Christ (<foreign xml:lang="gez">ዘደብረ፡ ታቦር፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4493Mary">
+               <label>Antiphon of Our Lady Mary (<foreign xml:lang="gez">ዘእግዝእትነ፡ ማርያም፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4494Abraham">
+               <label>Antiphon of Abraham (<foreign xml:lang="gez">ዘአብርሃም፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4495Melchizedek">
+               <label>Antiphon of Melchizedek (<foreign xml:lang="gez">ዘመልከ፡ ጼዴቅ፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4496Beheading">
+               <label>Antiphon of the beheading of John the Baptist (<foreign xml:lang="gez">ዘምትረተ፡ ርእሱ፡ ለዮሐንስ፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4497Libanos">
+               <label>Antiphon of ʾAbbā Libānos</label>
+            </div>
+            <div type="textpart" corresp="LIT4498Pilgrims">
+               <label>Antiphon of Pilgrims (<foreign xml:lang="gez">ዘፈላስያን፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4499Sunday">
+               <label>Antiphon of Sunday  (<foreign xml:lang="gez">ዘሰንበተ፡ ክርስቲያን፡</foreign>)</label>
+            </div>
+            <div type="textpart" xml:id="Haleta">
+              <label>Table of melodic variations for the Hallelujah <foreign xml:lang="gez">አንቀጽ፡ ሃሌታ፡</foreign></label>
```

Comment:
As far as I can see, the spelling is አንቀጸ፡ ሃሌታ (status constructus)

Reply:
You are quite correct - in the particular case I was looking at there is a spelling error - but it is indeed an error! I'll correct this in all three cases